basic_samples/Authentication/AuthorizationCodeFlow/DotNet/AuthorizationCodeFlow/LoopbackHttpListener.cs

```diff
@@ -17,7 +17,7 @@ public class LoopbackHttpListener : IDisposable
         public LoopbackHttpListener(int port, string path = null)
         {
             path ??= string.Empty;
-            if (path.StartsWith("/", StringComparison.OrdinalIgnoreCase)) path = path.Substring(1);
+            if (path.StartsWith("/", StringComparison.OrdinalIgnoreCase)) path = path[1..];
```

Comment:
repeat of earlier question

Reply:
Repeat of earlier answer 😄 
This came up as an information message from the build, so I accepted Visual Studio's suggestion to use the new format.

Reply:
i guess i should accept the new sugar and embrace the future (current)